### PR TITLE
Disable 3DTILES_multiple_contents support from 3d-tiles-next branch

### PR DESCRIPTION
**Pull request by ptrgags**
The implementation of the `3DTILES_multiple_contents` extension is not ideal:

* This extension can only be used with implicit tiling if every content has the same `refine` strategy. In practice, most implicit tilesets we've seen use a mix of `REPLACE` and `ADD`.
* The implementation has some race conditions when it comes to request scheduling that were not completely resolved.
* We're still not sure if making a separate Multiple3DTileContent class was the right approach or if modifying  `Cesium3DTile` to allow multiple contents would be better in the long run.

This PR makes the following temporary changes:
* Remove the `Multiple3DTileContent` class
* Disable `3DTILES_multiple_contents` extension in implicit tiling. Removing it completely would be much more work.

This way, there's one less detail holding up our staging branch `3d-tiles-next` from being merged into `master`.

We will revisit this functionality in the future, though this is low on our list of priorities for 3D Tiles next.

@lilleyse could you review?

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @ptrgags!
* :heavy_check_mark: Signed CLA found.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

File: Source/Scene/ImplicitTileset.js [hunk 1]
```diff
@@ -213,7 +214,11 @@ export default function ImplicitTileset(tileset, baseResource, tileJson) {
  */
 function gatherContentHeaders(tileJson) {
   if (has3DTilesExtension(tileJson, "3DTILES_multiple_contents")) {
-    return tileJson.extensions["3DTILES_multiple_contents"].content;
+    // This extension is disabled temporarily as the specification and Cesium3DTile implementation need revision.
+    throw new DeveloperError(
+      "3DTILES_multiple_contents is not currently supported"
+    );
+    //return tileJson.extensions["3DTILES_multiple_contents"].content;
```
Comment (lilleyse) [inline Source/Scene/ImplicitTileset.js:221]: Delete?

**Review by lilleyse — COMMENTED** [review]
I did a global search for `multiple_` and there are still some mentions of `3DTILES_multiple_contents` in public doc. There are also some test files with the extension, not sure if they are needed.

Can `MULTIPLE_CONTENT` be removed from `Cesium3DTileContentType.js`?

We talked offline about leaving the implicit files alone because it would difficult to extract out the multiple contents code. I forget, have there been significant changes to implicit tiling after multiple contents was merged? If not we could copy files from a previous commit.

**Comment by lilleyse** [comment]
> We talked offline about leaving the implicit files alone because it would difficult to extract out the multiple contents code. I forget, have there been significant changes to implicit tiling after multiple contents was merged? If not we could copy files from a previous commit.

Or is it possible to revert https://github.com/CesiumGS/cesium/pull/9407 and https://github.com/CesiumGS/cesium/pull/9432 cleanly?

**Comment by ptrgags** [comment]
@lilleyse 

>I forget, have there been significant changes to implicit tiling after multiple contents was merged?

There's been a few changes:

* adding Implicit tile metadata
* I forget if this was in the multiple contents PRs or afterwards, but there were some tests

>Or is it possible to revert #9407 and #9432 cleanly?

Not very cleanly. I attempted to revert #9432 in a separate local branch, but running into quite a few merge conflicts. Some were easy, but things like the `ImplicitTilingTester` looks like it would be a bit of work to untangle. Not sure if #9407 would introduce more conflicts, but I wouldn't be surprised since both PRs were merged about 5 weeks ago.

So both avenues seem a bit messy, not sure which is most worthwhile to spend time on.

**Comment by ptrgags** [comment]
Today I tried multiple contents again, I could not reproduce the problem I had seen weeks ago, even after zooming rapidly in and out.

In light of this and how involved it is to remove this temporarily, @lilleyse and I decided offline to close this PR and keep the code around as an experimental feature.

**Pull request closed without merging**